### PR TITLE
Upgrade module to SS4

### PR DESCRIPTION
**Pull request by StephenMakrogianni**
AC: 
- Installs correctly on SS ^4.2
- Accordions are displaying correctly
- Custom AccordionItems.ss template works
- Check the black list works

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,18 +3,14 @@ language: php
 sudo: false
 
 php:
- - 5.5
  - 5.6
```
Comment (Firesphere) [inline .travis.yml]: Please add PHP7

File: composer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ninja-unicorns/wysiwyg-accordion",
+  "name": "ninjaunicorns/wysiwygaccordion",
```
Comment (Firesphere) [inline composer.json]: Don't change the name, it will break composer and packagist

**Review by Firesphere — CHANGES_REQUESTED** [review]

File: Readme.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The items can be ordered using drag-and-drop on the gridfield overview.
 
 If you don't want the accordion option on certain pages, add the blacklisted pagetypes to your yml like this:
 ```yaml
-AccordionPageExtension:
+NinjaUnicorns\WysiwygAccordion\Extensions\AccordionPageExtension:
   PageBlacklist:
     - HomePage
```
Comment (scott-nz) [inline Readme.md]: Blacklisted pages need to be namespaced.

File: composer.json [hunk 4]
```diff
@@ -1,33 +1,47 @@
...
+        "silverstripe",
+        "Shortcode",
+        "Accordion",
+        "Ninja",
+        "Unicorns"
+    ],
+    "config": {
+        "process-timeout": 900,
+        "platform": {
+            "php": "5.6"
```
Comment (Firesphere) [inline composer.json]: php 7 is supported

Comment (StephenMakrogianni) [reply composer.json]: Yes. I'll add that in. Good spotting mate.

**Review by Firesphere — APPROVED** [review]

**Pull request merged**